### ISSUE TITLE
Permalink invalid Size

### ISSUE THREAD

**Issue by Naleesha** [dtube/production#33]
Hello,
for a few days now I can't upload any Videos to my dtube channel anymore.
I have three videos online by now. On Wednesday I tried to upload the next one. But when everything was ready, uploaded an all, i clicked on the "send" button. And it said: "failure Permalink invalid size".
I then tried with other Videos and other thumbnails, it was all the same issue.
I haven't done anything different than when I uploaded my first three videos, but it keeps saying the permalink issue.

How can I fix this and what do I have to do?

**Comment by skzap** [comment]
Try to log out and in again. Maybe clear your browser's cached data. Let me know what did it, I'm unsure about how this 'permalink' error coming back from the blockchain happens.

**Comment by Naleesha** [comment]
I found it. It was neither to log out and in again (that was the first thing I tried before opening this section) nor to delete trhe browser cache.

The issue was stated in the Tags. I used the Tags "Schottland" "Vlog" "Naleesha" and "#deutsch" for all videos wich were made in Scotland.
for some reasons, the IPFS couldn't resolve the "#" properly. I uploaded without any Tags and it worked. after confirmation I added the Tags by editing. I hope this answer is helpful for other People with similar Issues.
